scripts/ci/setup-dev-sso.sh

```diff
@@ -26,13 +27,20 @@ read_arguments() {
                     DEV_SSO_NS=$1
                     shift
                     ;;
+                -r|--rosa-cluster)
+                    shift
+                    CLUSTER_NAME="$1"
```

Comment:
I'd suggest adding prefix `ROSA` to this variable

```suggestion
                    ROSA_CLUSTER_NAME="$1"
```

Reply:
good point. Done in https://github.com/codeready-toolchain/toolchain-cicd/pull/116/commits/9671bdba68d5beb7a8404d9235d4c86b655fd1eb